controller/src/conf/logback.xml

```diff
@@ -18,16 +18,14 @@
 
     <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
         <file>controller-server.log</file>
-        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+        <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
```

Comment:
Since `SizeAndTimeBasedFNATP` is now deprecated, changed to `SizeAndTimeBasedRollingPolicy` that also allows specifying a cap on size of all log files put together.